config/servtd_info.json

```diff
@@ -1,6 +1,6 @@
 {
     "attributes":  "0000000000000000",
-    "xfam":  "e71a060000000000",
+    "xfam":  "e718060000000000",
```

Comment:
Will this change impact Intel's bare-metal TDX CI/CD pipeline run? That pipeline is triggered on all PR to intel/migtd repo.